rosdep/base.yaml

```diff
@@ -3358,6 +3358,18 @@ pcre-dev:
   ubuntu: [libpcre++-dev]
 phantomjs:
   ubuntu: [phantomjs]
+php:
+  debian:
+    jessie: [php5]
+  ubuntu:
+    precise: [php5]
+    raring: [php5]
+    saucy: [php5]
+    trusty: [php5]
+    utopic: [php5]
+    vivid: [php5]
+    wily: [php5]
+    xenial: [php]
```

Comment:
@kmhallen thanks for these updates. We also try to keep Fedora rules in sync: like here: https://admin.fedoraproject.org/pkgdb/package/rpms/php/

And looking at debian it looks like php7.0 is available on stretch. And like xenial php is provided by php7.0 http://packages.ubuntu.com/xenial/php7.0

so the rules for xenial and stretch should be php7.0 not just php.

Also would you mind adding yakkety and zesty?